web/tsconfig.json

```diff
@@ -19,7 +19,10 @@
       }
     ],
     "paths": {
-      "~/*": ["./src/*"]
+      "~/*": ["./src/*"],
+      "@contrak/db": ["../packages/db/src/sqlite/index.ts"],
```

Comment:
This alias is what allows the web app to specify that it wants to use sqlite.

Ideally this could be dynamic based on the `DATABASE_TYPE` env var. However, in order for typescript to be happy, this needs to be defined statically.